sdk/communication/azure-communication-rooms/src/main/java/com/azure/communication/rooms/RoomsAsyncClient.java

```diff
@@ -538,7 +541,7 @@ private CommunicationRoom getCommunicationRoomFromResponse(RoomModel room) {
      * @return The create room request.
      */
     private CreateRoomRequest toCreateRoomRequest(OffsetDateTime validFrom, OffsetDateTime validUntil,
-            Iterable<RoomParticipant> participants) {
+            boolean isPstnDialOutEnabled, Iterable<RoomParticipant> participants) {
```

Comment:
We should use `Boolean` as the `CreateRoomOptions` allows this value to be `null`.

```suggestion
            Boolean isPstnDialOutEnabled, Iterable<RoomParticipant> participants) {
```